### PR TITLE
CI/BLD: switch to entry points for scripts

### PR DESCRIPTION
**Pull request by klauer**
cc @ZLLentz  @hhslepicka

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,16 +8,18 @@ source:
   path : ..
 
 build:
-  number: 1
+  number: 2
   noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - lucid = lucid.launcher:main
```
Comment (hhslepicka) [inline recipe/meta.yaml]: ooops... some leftovers here from the lucid recipe

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,16 +8,18 @@ source:
...
+    - QSGenerateExcelSpreadSheet.py = psdm_qs_cli.QSGenerateExcelSpreadSheet:main
+    - QSGenerateJSON.py = psdm_qs_cli.QSGenerateJSON:main
 
 requirements:
   build:
     - python
-    - setuptools
-    - krtc
-    - requests
-    - six
```
Comment (hhslepicka) [inline recipe/meta.yaml]: Maybe those should go as `run` dependencies?

File: setup.py [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+    url="https://github.com/slaclab/psdm_qs_cli.git",
+    author="Murali Shankar",
+    author_email="mshankar@slac.stanford.edu",
+    license="MIT",
+    packages=["psdm_qs_cli"],
+    entry_points={
+        "console_scripts": [
+            "QSGenerateExcelSpreadSheet.py=psdm_qs_cli.QSGenerateExcelSpreadSheet:main",
+            "QSGenerateJSON.py=psdm_qs_cli.QSGenerateJSON:main",
+        ],
```
Comment (hhslepicka) [inline setup.py]: If we could track down who is using the scripts we could probably switch them to remove the `.py` at the name? 
This is just aesthetics... feel free to ignore

File: recipe/meta.yaml (hunk 1 above)
Comment (klauer) [reply recipe/meta.yaml]: D'oh - thanks for catching that 😄

File: recipe/meta.yaml (hunk 2 above)
Comment (klauer) [reply recipe/meta.yaml]: They are - the github diff just hides it though

File: setup.py (hunk 3 above)
Comment (klauer) [reply setup.py]: Yeah, I'd consider this back-compat. No clue who is using these.

Comment (hhslepicka) [reply setup.py]: Yeah, we better keep the same name! 😉

File: recipe/meta.yaml (hunk 2 above)
Comment (hhslepicka) [reply recipe/meta.yaml]: Oh, ok! Sorry about that.

**Comment by klauer** [comment]
Done, thanks!

**Pull request merged**